src/ProgramBlockEditor.js

```diff
@@ -174,22 +252,52 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
         return (
             <React.Fragment key={programStepNumber}>
                 <div className='ProgramBlockEditor__program-block-connector'/>
-                {this.makeProgramBlock(programStepNumber, command)}
+                {command !== 'addNode' ?
```

Comment:
Using a special command name here to select between (connector-AddNode-connector-tile) and (connector-AddNode) is unclear and fragile (for example, a user could create a custom tile called "addNode"). I would instead suggest making the difference explicit with separate functions:

- `makeProgramBlockSection` and
- `makeEndOfProgramAddNodeSection`